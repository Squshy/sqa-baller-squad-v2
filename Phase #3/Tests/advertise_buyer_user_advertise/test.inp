LOGIN
BUYINGMAN
ibuythings
ADVERTISE
logout
exit