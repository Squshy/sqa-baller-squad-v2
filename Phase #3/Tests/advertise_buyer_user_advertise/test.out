=========================================================
                        ADVERTISE
=========================================================

          Your account is a Buy Standard account.  
          You are notpermitted to auction items.

---------------------------------------------------------
=========================================================